tests/json/test_load_from_json_utils.py

```diff
@@ -31,31 +26,18 @@ def test_GIVEN_nx_class_in_different_formats_WHEN_reading_class_information_THEN
     assert _find_nx_class(class_attribute) == "NXmonitor"
 
 
-@pytest.mark.parametrize(
-    "test_input",
-    (
-        {},
-        {
-            "module": "dataset",
-            "config": {"values": 0},
-        },
-        {"attributes": []},
-    ),  # noqa E231
-)
-def test_GIVEN_empty_dictionary_or_dictionary_with_no_attributes_WHEN_adding_attributes_THEN_returns_nothing(
-    test_input,
-):
+def test_GIVEN_empty_dictionary_or_dictionary_with_no_attributes_WHEN_adding_attributes_THEN_returns_nothing():
     dataset = Dataset(parent_node=None, name="ds", values=123, type=ValueTypes.INT)
-    _add_attributes(test_input, dataset)
     assert not dataset.attributes
 
 
 def test_GIVEN_dictionary_containing_attribute_WHEN_adding_attributes_THEN_attribute_object_is_created():
     key = "units"
     value = "m"
-    test_dict = {"attributes": [{"name": key, "values": value}]}
     dataset = Dataset(parent_node=None, name="ds", values=123, type=ValueTypes.INT)
-    _add_attributes(test_dict, dataset)
+    attributes = Attributes()
```

Comment:
this test was supposed to test `_add_attributes` function. Since you removed that, I am not sure if this test is required. In case you want to test adding Attributes, this may go in test for Dataset class itself. What do you think?

Reply:
Same for other modified tests in this PR.

Reply:
So I was thinking just to keep it like this to test attributes creation separately (to make it more "unit" like).

Reply:
But it might make sense to move the test elsewhere

Reply:
Yes, because even the name of test is now misleading.